src/lines-component.coffee

```diff
@@ -19,11 +20,7 @@ class LinesComponent
   placeholderTextDiv: null
 
   constructor: ({@presenter, @hostElement, @useShadowDOM, visible}) ->
```

Comment:
@nathansobo: I experimented with 2) and apparently we're not saving a significant amount of time. Moreover, batching HTML parsing requires us to add stuff in a two-step fashion: namely, HTML accumulation and rendering. Not a huge boost in complexity, but neither a "free lunch" (TANSTAAFL), therefore I'd say it's not worth going further down that path.

As for 3) it seems like we may be able to save ~1ms if we do this for gutter as well. What about addressing it after merging this?